allel/stats/roh.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+from hmmlearn import hmm
+from allel.chunked import ChunkedArray
+
+
+def call_roh(g, pos, ix, emission=(0.001, 0.0025, 0.01),
+             transition=1e-6, min_roh=100000,
+             is_accessible=None, contig_size=None):
+
+    """
+    Call ROH (runs of homozygosity) of a given individual
+
+    Parameters
+    ----------
+    g : array_like, int, shape (n_variants, n_samples, ploidy)
+        Genotype array.
+    pos: array_like, int, shape (n_variants)
+        Positions of variants, same 0th dimension as g
+    ix: integer
+        index of the sample on which to call roh
+    emission: tuple or array_like, float
+        Probability of observing a heterozygote in each hidden state.
+        Supports >= 2 states. Appropriate values will depend on
+        frequency of sequencing error & spurious het calls due
+        to misalignment
+    transition: float
+        Probability of moving between states
+    min_roh: integer
+        Minimum size (bp) to condsider as a ROH. Will depend on contig size
+        and recombination rate. In humans typically > 1Mbp
+    is_accessible: array_like, bool, shape (npositions)
+        Boolean array for each locus in contig describing whether accessible
+        or not.
+    contig_size: int
+        If is_accessible not known/not provided, allows specification of
+        total length of contig.
+
+    Returns
+    -------
+    roh: ndarray, int, shape (n_roh, 2)
+        Span windows of ROH
+    froh: float
+        Proportion of genome in ROH
+
+    """
+
+
+    # this function computes the likely ROH using an HMM model.
+    # define model:
+    ### 2 state model, 3 observable
+    # 0: ROH
+    # 1: normal
+    # 2: inaccessible
```

Comment:
Can we move the explanation of the model to a Notes section within the docstring, and provide a little bit more explanation of what the two hidden states and 3+ emission states are?